Makefile

```diff
@@ -16,4 +16,10 @@ docker-run:
 	layer5/meshery-istio
 
 run:
-	DEBUG=true GOPROXY=direct GOSUMDB=off go run main.go
\ No newline at end of file
+	DEBUG=true GOPROXY=direct GOSUMDB=off go run main.go
+
+test: 
+	brew install nektos/tap/act
```

Comment:
Also this will run all of the gh actions on local but we just need to run tests .

Reply:
@gunjanmimo what do we want to do with this PR?

Reply:
@leecalcote sir, running git action locally helps in some cases, but my this pr is not showing all the details properly.

Reply:
@gunjanmimo if you'll sync your fork and update this PR to only include changes to the Makefile, we can get this PR merged quickly.

Reply:
@gunjanmimo if you'll sync your fork and update this PR to only include changes to the Makefile, we can get this PR merged quickly.